apps/arm/ClawController.cpp

```diff
@@ -137,6 +137,7 @@ float ClawController::convertGapCmToShaftPositionDegrees(float gap_cm) {
     // return (8.282382533e-3)*(gap_cm*gap_cm*gap_cm*gap_cm) - (2.986760459e-1)*(gap_cm*gap_cm*gap_cm) + (5.007842722)*(gap_cm*gap_cm) - (171.560244)*gap_cm + 2127.848743; // High precision 
 }
 
+
 float ClawController::convertGapVelocityCmToShaftVelocityDegrees(float gap_cmPerSec) {
     return 2*(1.573564198)*gap_cmPerSec - 158.4968661;
```

Comment:
Probably not the most important things but when I see these constants I have no context as to how you decided to use these values and what they convert.

Reply:
use constants for these magic numbers: https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
I rebased with alex's arm code at some point early on (I think), so thats why I have all this added. Not actually my code.

Reply:
Yes this is my code from SAR grind. Will clean at some point but not Tony's job.